src/app/auth/services/auth.service.ts

```diff
@@ -188,31 +189,25 @@ export const changePassword = async (newPassword: string, currentPassword: strin
   const privateKey = Buffer.from(user.privateKey, 'base64').toString();
   const privateKeyEncrypted = aes.encrypt(privateKey, newPassword, getAesInitFromEnv());
 
-  const response = await fetch(`${process.env.REACT_APP_API_URL}/api/user/password`, {
-    method: 'PATCH',
-    headers: httpService.getHeaders(true, true),
-    body: JSON.stringify({
-      currentPassword: encryptedCurrentPassword,
-      newPassword: encryptedNewPassword,
-      newSalt: encryptedNewSalt,
-      mnemonic: encryptedMnemonic,
-      privateKey: privateKeyEncrypted,
-    }),
-  });
-
-  const data = await response.json();
-
-  if (response.status === 500) {
-    analyticsService.track(email, 'error');
-    throw new Error('The password you introduced does not match your current password');
-  }
-
-  if (response.status !== 200) {
-    analyticsService.track(email, 'error');
-    throw new Error(data.error);
-  }
+  const usersClient = createUsersClient();
 
-  analyticsService.track(email, 'success');
+  return usersClient.changePassword(<ChangePasswordPayload>{
```

Comment:
Be careful, this is not working as intended. Changing the password with this changes corrupts something involved with the mnemonic, and the user won't be able to regenerate his/her old mnemonic nor download any file uploaded before changing the password. 

Steps for test it: 
1- Signup
2- Upload a file, a pdf for example, that gets corrupted with any minimal change.
3- Change password
4- Logout
5- Login
6- Download the previously uploaded file

Reply:
@sg-gs I just tried again ad it works fine here :/ What error are you having?

Reply:
Since we already found what was the error here I'm closing this.

Did you review all the code or stopped when found this?